images/scripts/fedora.setup

```diff
@@ -220,8 +240,10 @@ rm -rf /var/log/audit/
 # clean up old kernels after upgrading
 dnf repoquery --installonly --latest-limit=-1 -q | xargs --no-run-if-empty rpm -e --verbose
 
-# pmlogger.service sometimes causes long shutdown hangs; disable all PCP services and timers
-systemctl --all --legend=false list-units 'pm*' | awk '{print $1}' | xargs systemctl disable
+if [ "fedora-eln" != "$IMAGE" ]; then
+    # pmlogger.service sometimes causes long shutdown hangs; disable all PCP services and timers
+    systemctl --all --legend=false list-units 'pm*' | awk '{print $1}' | xargs systemctl disable
+fi
```

Comment:
Actually.. [that recent build failure](https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/fedora-eln-20231129-190101.log) is exactly the reason why we need this hack. Please put it back for ELN. In fact, it seems it needs to become stronger, and append a `--now`.

Reply:
I had it in place and if I recall there were not units matched, making xargs run with no arguments and fail.
Retrying it again.

Reply:
Nice, thank you!